README.md

```diff
@@ -840,7 +843,40 @@ clientConfig.GetSerializationConfig()
 
 From now on, Hazelcast will use `CustomSerializer` to serialize `CustomSerializableType` objects.
 
-## 4.4. Global Serialization
+## 4.4. JSON Serialization
+
+You can use the JSON formatted strings as objects in Hazelcast cluster. Starting with Hazelcast IMDG 3.12, the JSON serialization is one of the formerly supported serialization methods. Creating JSON objects in the cluster does not require any server side coding and hence you can just send a JSON formatted string object to the cluster and query these objects by fields.
+
+In order to use JSON serialization, you should use the `HazelcastJsonValue` object for the key or value. Here is an example IMap usage:
+
+```c#
+    var config = new ClientConfig();
+    var client = HazelcastClient.NewHazelcastClient(config);
+    var map = client.GetMap<string, HazelcastJsonValue>("map");
+```
+
+We constructed a map in the cluster which has `string` as the key and `HazelcastJsonValue` as the value. `HazelcastJsonValue` is a simple wrapper and identifier for the JSON formatted strings. You can get the JSON string from the `HazelcastJsonValue` object by using the `JsonString` property. 
+
+You can construct a `HazelcastJsonValue` using one of the constructors. All constructors accept the JSON formatted string as the parameter. No JSON parsing is performed but it is your responsibility to provide correctly formatted JSON strings. The client will not validate the string, and it will send it to the cluster as it is. If you submit incorrectly formatted JSON strings and, later, if you query those objects, it is highly possible that you will get formatting errors since the server will fail to deserialize or find the query fields.
```

Comment:
We don't provide constructors, there is only one static Factory method. This is different from C++

Reply:
Fixed. Strange. I was sure I changed the fragment about constructors already...